maro/simulator/scenarios/vm_scheduling/topologies/azure.2019.10k/config.yml

```diff
@@ -23,22 +23,105 @@ MAX_MEM_OVERSUBSCRIPTION_RATE: 1
 # Max CPU utilization rate.
 MAX_UTILIZATION_RATE: 1
 
-PM:
-  - PM_type: 0 # PM type is currently "int" only.
-    amount: 100
-    CPU: 32
-  # GPU: 0
-    memory: 128
-    power_curve:
-      calibration_parameter: 1.4
-      busy_power: 10
-      idle_power: 1
-# - PM_type: 1
-#   amount: 50
-#   CPU: 32
-#   GPU: 0
-#   memory: 128
-#   power_curve:
-#     calibration_parameter: 1.4
-#     busy_power: 10
-#     idle_power: 1
+components:
+  pm:
+    - pm_type: 0
+      cpu: 32
+      memory: 128
+      power_curve:
+        calibration_parameter: 1.4
+        busy_power: 10
+        idle_power: 1
+    - pm_type: 1
```

Comment:
pm_type 1 seems same to pm_type 0

to illustrate how to set different pm types, you can change the cpu/memory/power of type 1?